### PR TITLE
Implement Shed Tail

### PR DESCRIPTION
**Pull request by Koekenpann**

### CODE REVIEW

File: data/text/197.txt [hunk 1]
```diff
@@ -1397,3 +1397,28 @@ The weirdness disappeared\nfrom the battlefield.
...
+The foe’s {STRVAR_1 1, 0, 0} is preparing\nto tell a chillingly bad joke!
+{STRVAR_1 1, 0, 0} shed its tail to\ncreate a decoy!
+The wild {STRVAR_1 1, 0, 0} shed its tail to\ncreate a decoy!
+The foe’s {STRVAR_1 1, 0, 0} shed its tail to\ncreate a decoy!
+{STRVAR_1 1, 0, 0} is overflowing\nwith space power!
+The wild {STRVAR_1 1, 0, 0} is overflowing\nwith space power!
+The foe’s {STRVAR_1 1, 0, 0} is overflowing\nwith space power!
+{STRVAR_1 1, 0, 0} absorbed\nelectricity!
+The wild {STRVAR_1 1, 0, 0} absorbed\nelectricity!
+The foe’s {STRVAR_1 1, 0, 0} absorbed\nelectricity!
```
Comment (BluRosie) [inline data/text/197.txt:1424]: few things here:

no need for new line at 1400
and are these all lines for other moves that are going on?  i recognize chilly reception but am not sure about the other ones

File: src/moves.c [hunk 2]
```diff
@@ -171,6 +171,7 @@ u16 move_effect_to_subscripts[] =
     [ADD_STATUS_EFF_WORK_UP]                     = SUB_SEQ_WORK_UP,
     [ADD_STATUS_EFF_SPICY_EXTRACT]               = SUB_SEQ_SPICY_EXTRACT,
     [ADD_STATUS_EFF_FILLET_AWAY]                 = SUB_SEQ_FILLET_AWAY,
+	[ADD_STATUS_EFF_SHED_TAIL]                   = SUB_SEQ_HANDLE_SHED_TAIL,
```
Comment (BluRosie) [inline src/moves.c:174]: possibly convert this to spaces, i try to keep things consistent when i can

File: armips/move/battle_eff_seq/142.s [hunk 3]
```diff
@@ -10,6 +10,10 @@
 .create "build/move/battle_eff_seq/0_142", 0
 
 a030_142:
+    if IF_NOTEQUAL, VAR_CURRENT_MOVE, MOVE_SHED_TAIL, _BellyDrum 
+    changevar VAR_OP_SET, VAR_ADD_STATUS1, ADD_STATUS_WORK | ADD_STATUS_SHED_TAIL
+    endscript
+_BellyDrum:
```
Comment (BluRosie) [inline armips/move/battle_eff_seq/142.s:16]: is there a reason to build off of belly drum?  just curious, not sure if it's for the AI or what you're thinking

**Review by BluRosie — COMMENTED** [review]
got a few questions

**Comment by Koekenpann** [comment]
As Aero recommended, I have separated it to a different battle effect sequence (see: https://discord.com/channels/446824489045721090/810895757309771796/1212212759622717440)

**Comment by BluRosie** [comment]
thank u!

**Pull request merged**